README.md

```diff
@@ -18,6 +18,11 @@ Prerequisites
 
 This requires Go 1.4 or above.
 
+Design Constraints
+------------------
+
+The `google.golang.org/grpc` Go package must not depend on any external Go libraries. This is to simplify use when vendored into other Go projects.
```

Comment:
THat is a great way to put it.

On Mon, Sep 21, 2015 at 10:53 AM Qi Zhao <notifications@github.com> wrote:

> In README.md
> <https://github.com/grpc/grpc-go/pull/345#discussion_r40002547>:
>
> > @@ -18,6 +18,11 @@ Prerequisites
> >
> >  This requires Go 1.4 or above.
> >
> > +Design Constraints
> > +------------------
> > +
> > +The `google.golang.org/grpc` <http://google.golang.org/grpc> Go package must not depend on any external Go libraries. This is to simplify use when vendored into other Go projects.
>
> I meant https://github.com/golang.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc-go/pull/345/files#r40002547>.
>